### DISCUSSION TITLE
I would like to help adding more math functions

### DISCUSSION THREAD

**Discussion by cemremengu** [antonmedv/expr#708] [General]
I would like to contribute more math functions along the lines of [python's numexpr lib](https://github.com/pydata/numexpr/blob/7484828ac5e3454201169f81d5d90a13203d988d/doc/user_guide.rst#supported-functions).

What do you think about this? Where should I begin? Any pointers to getting started with the codebase is appreciated.

**Comment by antonmedv** [comment]
This is very nice! I think the first thing will be to add some sort of namespace for functions. And add all of those in `math.*` name space.
